### ISSUE TITLE
Refactor when moving files

### ISSUE THREAD

**Issue by claudius25** [Microsoft/vscode#26314]
The best feature you could have is to refactor when moving files in the explorer (out of the box) 
There is a extension (move ts) but it's buggy. 
It's a basic refactor thing, which to my surprise is not in visual code. 

So if i move a file from a folder to another, i want it's path reference to be updated automatically in the project. I tried this on a angular 2 project with .ts files and it did not work. I haven't tried with other files.

Thanks

**Comment by stringham** [comment]
I believe this is a duplicate of https://github.com/Microsoft/vscode/issues/19439 which is waiting on TS language support tracked by https://github.com/Microsoft/TypeScript/issues/6487.

Because I needed this functionality for a large project, I created the [Move TS](https://marketplace.visualstudio.com/items?itemName=stringham.move-ts) extension to get by in the meantime. If you're having issues with the extension, I recommend opening an issue: https://github.com/stringham/move-ts/issues

**Comment by mjbvz** [comment]
Duplicate of #19439
